doc/user_guide/io.rst

```diff
@@ -599,6 +599,22 @@ documentation for more information.
     >>> s.save('file.jpg', scalebar=True)
     >>> s.save('file.jpg', scalebar=True, scalebar_kwds={'location':'lower right'})
 
+In the example above, the image is created using
+:py:func:`~.matplotlib.pyplot.imshow`, and additional keyword arguments can be
+passed to this function using ``imshow_kwds``. For example, this can be used
+to save an image displayed using a matplotlib colormap:
+
+.. code-block:: python
+
+    >>> s.save('file.jpg', imshow_kwds=dict(cmap='viridis'))
+
+
+The number of pixels of the exported image can be adjusted:
+
+.. code-block:: python
+
+    >>> s.save('file.jpg', output_size=512)
```

Comment:
At least mention in the user guide (like in the docstring) that this case sets the width of the image (number of pixels sounds like the total number of pixels of the image is given).

Even better, explain the three possible values for `output_size`:
- tuple = (width,height)
- int = width
- None = original size of image

Reply:
Mention here or in the docstring that if a tuple is given and the aspect ratio is different from the original image, the additional area is padded with white pixels?